spec/lib/synapse/haproxy_spec.rb

```diff
@@ -1,16 +1,19 @@
 require 'spec_helper'
+require 'synapse/config_generator/haproxy'
```

Comment:
this spec file should probably live in `spec/lib/config_generator/haproxy_spec.rb`